src/main/java/com/devathon/preguntonic/dto/PlayerEvent.java

```diff
@@ -0,0 +1,11 @@
+/*
+ *
+ * Copyright (c) 2024 Devathon., All Rights Reserved.
+ *
+ */
+package com.devathon.preguntonic.dto;
+
+import lombok.Builder;
+
+@Builder
+public record PlayerEvent(BasicPlayer player, String event) {}
```

Comment:
where will be used this class? The events will have some metadata ??

Reply:
For now I added it for the events that each user has carried out in "Join", "Ready", "UnReady" (in RoomController `joinRoom`, `playerReadyInRoom` and `playerUnReadyInRoom`) to be able to send it to the same topic and in the front-end the event can be handled according to the `action-event` performed.